src/Microsoft.DotNet.Build.Tasks/PackageFiles/ilasm/ilasm.depproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <!-- suppress the attempt to copy build output. -->
+    <CopyBuildOutputToOutputDirectory>false</CopyBuildOutputToOutputDirectory>
+  </PropertyGroup>
+
+  <Target Name="CoreCompile">
+    <Copy SourceFiles="@(NativeCopyLocalItems)" DestinationFolder="$(MSBuildThisFileDirectory)" />
+  </Target>
+
+  <!-- Required by Common.Targets -->
+  <Target Name="CreateManifestResourceNames" />
+
+  <PropertyGroup>
+    <NuGetTargetMoniker>.NETCoreApp,Version=v2.0</NuGetTargetMoniker>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <RidSpecificAssets>true</RidSpecificAssets>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NETCore.Runtime.CoreCLR" Version="2.1.0-preview1-25916-01" />
```

Comment:
We need to be careful about hardcoding these versions in BuildTools. If the output for this is isolated it might be OK but if it isn't isolated it might conflict with the other tools. We also need a strategy about updating this version.

Reply:
Is this any different from the version of Roslyn hardcoded in `init-tools.cmd`? I can hardcode it in the same place as `ROSLYNCOMPILERS_VERSION` if that will make you feel better about this but it will be equally unconfigurable.

Reply:
If we move this logic to the il.targets we could control this via a property and we could have a default value for it but we having a way to override it would be good.